terraform/modules/ecs/input.tf

```diff
@@ -88,10 +84,6 @@ variable "update_pay_scale_task_command" {
   type = "list"
 }
 
-variable "update_vacancies_task_command" {
```

Comment:
Did we mean to remove vacancy updater?

Reply:
Yes this was the plan.

I've had a brief look in production and there don't seem to be any more scraped listings that require major changes + don't expire soon so I think there is no value in breaking up the PR.

Reply:
Oh this is not the index to google when vacancy update. Never mind!